config/manager/manager.yaml

```diff
@@ -48,9 +63,11 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 100Mi
+            memory: 1024Mi
           requests:
-            cpu: 100m
-            memory: 20Mi
+            cpu: 50m
```

Comment:
These values changed. Does it matter?

Reply:
Changed to match the values we have on /bundle/manifests/ because `make bundle` was changing it back to these.